app/src/main/java/com/stl/skipthelibrary/BorrowersBooksActivity.java

```diff
@@ -30,25 +33,35 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void getBooks() {
         //Currently just test data as firebase is empty
-        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+        books.add(new Book("test ISBN" ,new BookDescription("test Title", "test Sysnopsis",
                 "test author", new Rating()),"testUsername",
                 new RequestHandler(new State(BookStatus.ACCEPTED,null, null)), null, new Rating()));
-        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+        books.add(new Book("test ISBN", new BookDescription("test Title", "test Sysnopsis",
                 "test author", new Rating()),"testUsername",
                 new RequestHandler(new State(BookStatus.REQUESTED,null, null)), null, new Rating()));
-        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+        books.add(new Book("test ISBN", new BookDescription("test Title", "test Sysnopsis",
                 "test author", new Rating()),"testUsername",
                 new RequestHandler(new State(BookStatus.AVAILABLE,null, null)), null, new Rating()));
-        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+        books.add(new Book("test ISBN", new BookDescription("test Title", "test Sysnopsis",
                 "test author", new Rating()),"testUsername",
                 new RequestHandler(new State(BookStatus.BORROWED,null, null)), null, new Rating()));
-        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+        books.add(new Book("test ISBN", new BookDescription("test Title", "test Sysnopsis",
                 "test author", new Rating()),"testUsername",
                 new RequestHandler(new State(BookStatus.REQUESTED,null, null)), null, new Rating()));
     }
 
     private void initRecyclerView(){
-        BookRecyclerAdapter adapter = new BookRecyclerAdapter(this, books);
+
+        Query songQuery = FirebaseDatabase.getInstance()
```

Comment:
Why is this called songQuery? 
```suggestion
        Query bookQuery = FirebaseDatabase.getInstance()
```